cmip6_downscaling/data/cmip.py

```diff
@@ -59,13 +53,13 @@ def postprocess(ds: xr.Dataset) -> xr.Dataset:
 
 
 def load_cmip(
-    activity_ids: str = "CMIP",
-    experiment_ids: str = "historical",
-    member_ids: str = "r1i1p1f1",
-    source_ids: str = "MIROC6",
-    table_ids: str = "day",
-    grid_labels: str = "gn",
-    variable_ids: List[str] = ["tasmax"],
+    activity_ids: str,
+    experiment_ids: str,
+    member_ids: str,
+    source_ids: str,
+    table_ids: str,
+    grid_labels: str,
+    variable_ids: str,
```

Comment:
```suggestion
    activity_ids: str = None,
    experiment_ids: str = None,
    member_ids: str = None,
    source_ids: str = None,
    table_ids: str = None,
    grid_labels: str = None,
    variable_ids: list[str] = None,
```

I think we should still be defaulting to None so these are not positional arguments. Also `variable_ids` should be a list, right?

Reply:
if `variable_ids` is just a string, I suggest changing it to a `variable_id`.